### ISSUE TITLE
HTTPS vs SSH

### ISSUE THREAD

**Issue by powersa** [MaptimeCalgary/presentation-example#1]
The readme recommends https for `git clone` and ssh for `git remote set-url origin`. This took me a moment to figure out, not sure if it is intentional.

**Comment by alukach** [comment]
Yeah, that's a bit weird... I think the thought was that you needed SSH keys set up to clone with `ssh` but you'll probably want to use `ssh` with a key when you push... I dunno.

What about it stopped you?  Just, kind of a "that seems weird" type of thing?

**Comment by powersa** [comment]
I'd have to go back through it to get the specifics, but basically you clone through https and then set the remote with ssh... That caused a bit of trouble for me, so I ended up setting the remote with https.
